dask/array/tests/test_array_core.py

```diff
@@ -35,12 +35,7 @@
 from dask.array.utils import assert_eq, same_keys
 
 # temporary until numpy functions migrated
```

Comment:
Can this comment also be dropped? Looks like it's referring to the `try/except`

```suggestion
```

Reply:
Yep, thanks for catching that @jrbourbeau. Have committed this change.